mne/annotations.py

```diff
@@ -102,6 +103,15 @@ def __len__(self):
         """Return the number of annotations."""
         return len(self.duration)
 
+    def __add__(self, other):
+        """Add (concatencate) two Annotation objects."""
```

Comment:
you should also copy other

Reply:
`append` does not / should not modify any of those inputs in place so I don't think it's necessary

Reply:
Why? I'm not modifying `other`.

Reply:
indeed...